canonical/ubuntu-advantage-client
# Do not show beta services in autocomplete

## Proposed Commit Message
Do not show beta services in autocomplete

When running ua enable, we don't want to show beta services by default in when users hit tab and the command autocompletes. We are now updating the logic to only show non-beta services in that situation. We will only show beta services in autocomplete if the user has a uaclient.conf configuration that allow beta services.

Fixes: #1594

## Test Steps
1. Launch a vm with this version of uaclient installed
2. Try to autocomplete when running `ua enable`. Verify that only non-beta services appear
3. Add the following lines to `/etc/ubuntu-advantage/uaclient.conf`:

```yaml
features:
  allow_beta: true
```

4. Refresh bash with `exec bash`
5. Try to autocomplete when running `ua enable`. Verify that only all services appear now

## Desired commit type::


## Checklist:
